client/src/app/pages/dashboard/maps/map-list/map-list.component.html

```diff
@@ -32,6 +56,14 @@ <h3 class="m-0" *ngSwitchCase="mapListType.TYPE_UPLOADS">
       <i>You have not uploaded any maps. Try <a routerLink="/dashboard/maps/uploads/new">uploading a new one!</a></i></h3>
   </div>
 </div>
+<div class="text-center" *ngIf="maps.length === 0 && requestSent && isSearchFiltered()">
+  <div [ngSwitch]="type">
+    <h3 class="m-0" *ngSwitchDefault>No maps with those search parameters found.</h3>
+    <h3 class="m-0" *ngSwitchCase="mapListType.TYPE_LIBRARY">No maps with those search parameters found in your library.</h3>
+    <h3 class="m-0" *ngSwitchCase="mapListType.TYPE_FAVORITES">No favorite maps with those search parameters found.</h3>
+    <h3 class="m-0" *ngSwitchCase="mapListType.TYPE_UPLOADS">You have not uploaded any maps with those search parameters.</h3>
```

Comment:
This should happen in the component backing (Typescript) code, since the type of list is only set once.

Reply:
I refactored it into the component but there's a similar switch statement in the template file right before it here: https://github.com/momentum-mod/website/blob/897aa1264323f446dea7af966b26b8231ef0ef40/client/src/app/pages/dashboard/maps/map-list/map-list.component.html#L51-L57

Should I refactor that out as well?

Reply:
Okay never mind I just realized it would require bypassing the sanitizer and would ultimately be messier.

Reply:
Yeah it gets more complicated when there's another element in the middle. Typically if it's just text, though, you can definitely using a component variable binding.